test/python/primitives/test_estimator.py

```diff
@@ -75,7 +75,7 @@ def test_estimator_run(self):
         # Note that passing objects has an overhead
         # since the corresponding indices need to be searched.
         # User can append a circuit and observable.
-        # calculate [ <psi2(theta2)|H2|psi2(theta2)> ]
+        # calculate [ <psi2(theta2)|H1|psi2(theta2)> ]
         result2 = estimator.run([psi2], [hamiltonian1], [theta2]).result()
```

Comment:
H1 should be H2 because the test uses `hamiltonian1`. Same for other estimator tests.